### DISCUSSION TITLE
Decompiler view?

### DISCUSSION THREAD

**Discussion by nullsection** [Vector35/binaryninja-api#2274] [Q&A] [answered]
I'm not too sure how to access the decompiled code. Is this just the HIL tab? 

I have the personal edition. In Ghidra it just shows the decompiled code to the right, is there a hotkey?

Thanks.

**Comment by psifertex** [comment, marked as answer]
Sorry for the delay in responding to this. Yes, HLIL is our decompiler view. If you would like to show them side-by-side, just use the "View / Native Docks / Show Reflection" menu. You can find more details on this in our latest blog post: 

https://binary.ninja/2021/02/24/2.3-release.html#view-synchronization
